### PR TITLE
Add OIE rate limits

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Add OIE rate limits information
- **Is this PR related to a Monolith release?** TBD

### Resolves:

* [OKTA-386423](https://oktainc.atlassian.net/browse/OKTA-386423)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 1]
```diff
@@ -49,7 +49,10 @@ These rate limits apply to all new Okta organizations. For orgs created before 2
 
 ### End-user rate limits
 
-Okta limits the number of requests from the Admin Console and End-User Dashboard to 40 requests per user per 10 seconds per endpoint. This rate limit protects users from each other and from other API requests in the system.
+Okta limits the number of requests:
+* From the Admin Console and End-User Dashboard, to 40 requests per user per 10 seconds per endpoint. This rate limit protects users from each other and from other API requests in the system.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:53]:
```suggestion
* From the Admin Console and the End-User Dashboard, to 40 requests per user per 10 seconds per endpoint. This rate limit protects users from each other and from other API requests in the system.
```

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 2]
```diff
@@ -36,6 +36,14 @@ This page provides the API rate limits for authentication and end-user activitie
 | `/api/{apiVersion}/radius`                                                                                             | 100              | 600              | 600     | 600        | 600                   |
 | `/idp/idx`                                                                                                             | 100              | 600              | 600     | 600        | 500                   |
 | `/login/token/redirect`                                                                                                             | 100              | 600              | 600     | 600        | 600                   |
+| <ApiLifecycle access="ie" /> **Identity Engine rate limit buckets:**                                                                            |               |               |      |         |                    |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:39]: Should we use "buckets" here? Perhaps use "requests" or another category term?

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
Looks good. Just a nit.

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md (hunk 2 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:39]: I replaced "buckets" with "requests".

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 3]
```diff
@@ -36,6 +36,14 @@ This page provides the API rate limits for authentication and end-user activitie
 | `/api/{apiVersion}/radius`                                                                                             | 100              | 600              | 600     | 600        | 600                   |
 | `/idp/idx`                                                                                                             | 100              | 600              | 600     | 600        | 500                   |
 | `/login/token/redirect`                                                                                                             | 100              | 600              | 600     | 600        | 600                   |
+| <ApiLifecycle access="ie" /> **Identity Engine rate limit requests:**                                                                            |               |               |      |         |                    |
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:39]: "Identity Engine rate limit requests:"
->
"Okta Identity Engine APIs:"

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 4]
```diff
@@ -36,6 +36,14 @@ This page provides the API rate limits for authentication and end-user activitie
...
+| `/idp/idx`                                                                                                     | 100              | 600              | 600     | 600        | 1,200                   |
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:40]: I'd recommend re-ordering these rows to:
```
/idp/idx
/idp/idx/identify
/idp/idx/introspect
/idp/idx/challenge/poll
/idp/idx/authenticators/poll
/idp/idx/poll
```

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 5]
```diff
@@ -49,7 +49,10 @@ These rate limits apply to all new Okta organizations. For orgs created before 2
 
 ### End-user rate limits
 
-Okta limits the number of requests from the Admin Console and End-User Dashboard to 40 requests per user per 10 seconds per endpoint. This rate limit protects users from each other and from other API requests in the system.
+Okta limits the number of requests:
+* From the Admin Console and the End-User Dashboard, to 40 requests per user per 10 seconds per endpoint. This rate limit protects users from each other and from other API requests in the system.
+
+* From the Identity Engine, to 20 requests per user per 5 seconds and 10 requests per state token per 5 seconds. <ApiLifecycle access="ie" />
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:55]: "To the Okta Identity Engine"

Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:55]: Fixed

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md (hunk 3 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:39]: Fixed, however, can't use "Okta Identity Engine", have been told to use "Identity Engine".

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md (hunk 4 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:40]: Fixed

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 6]
```diff
@@ -36,6 +36,14 @@ This page provides the API rate limits for authentication and end-user activitie
 | `/api/{apiVersion}/radius`                                                                                             | 100              | 600              | 600     | 600        | 600                   |
 | `/idp/idx`                                                                                                             | 100              | 600              | 600     | 600        | 500                   |
 | `/login/token/redirect`                                                                                                             | 100              | 600              | 600     | 600        | 600                   |
+| <ApiLifecycle access="ie" /> **Identity Engine APIs:**                                                                            |               |               |      |         |                    |
+| `/idp/idx`                                                                                                     | 100              | 600              | 600     | 600        | 1,200                   |
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:40]: shouldn't this last column also be a value of 600?

Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md]: Fixed.

**Review by vanngo-okta — APPROVED** [review]
LGTM

**Pull request merged**